core/Concurrency/LockBackend/MySqlLockBackend.php

```diff
@@ -0,0 +1,165 @@
+<?php
+/**
+ * Piwik - free/libre analytics platform
+ *
+ * @link http://piwik.org
+ * @license http://www.gnu.org/licenses/gpl-3.0.html GPL v3 or later
+ *
+ */
+
+namespace Piwik\Concurrency\LockBackend;
+
+
+use Piwik\Common;
+use Piwik\Concurrency\LockBackend;
+use Piwik\Db;
+use Piwik\DbHelper;
+
+class MySqlLockBackend implements LockBackend
```

Comment:
Might be good to write some simple tests for this backend as well? Can be probably copied from Queued tracking

Reply:
Ah yes, I forgot about the backend, I'll add the tests.